### DISCUSSION TITLE
Find maximum value of a column and return it's corresponding row values.

### DISCUSSION THREAD

**Discussion by Abwujki** [community/community#31123] [Other Feature Feedback, Questions, & Ideas]
Find maximum value of a column and return it's corresponding row values in python pandas

**Comment by emploiyer** [comment]
ok

**Comment by jfullstackdev** [comment]
in what language you are asking it to be implemented ? 

you can also check related coding problems here 

https://github.com/jdevstatic/java-coding-problems

**Comment by Abwujki** [comment]
I want it in python pandas

**Comment by murilochaves** [comment]
You can resolv using `.idmax()` and `.loc`:

```
import pandas as pd
import numpy as np

df = pd.DataFrame(
    np.random.randn(5, 3),
    columns=['column_01','column_02','column_03']
)
display(df)

df.loc[df['column_01'].idxmax(),]
```

And, access all columns using `.name_column`:

`df.loc[df['column_01'].idxmax(),].column_01
`
